examples/twitter/tweet_at_cozmo.py

```diff
@@ -182,20 +182,22 @@ def get_command(self, command_name):
 
     def extract_command_from_string(self, in_string):
```

Comment:
This should be renamed to pluralize commands (e.g. to extract_commands_from_string) if it now returns multiple commands.